ui/src/components/user.tsx

```diff
@@ -570,6 +572,21 @@ export class User extends Component<any, UserState> {
                   />
                 </div>
               </div>
+              <div class="form-group row">
+                <label class="col-lg-3 col-form-label" htmlFor="user-bio">
+                  {i18n.t('bio')}
+                </label>
+                <div class="col-lg-9">
+                  <input
+                    id="user-bio"
+                    class="form-control"
+                    placeholder={i18n.t('optional')}
+                    value={this.state.userSettingsForm.bio}
+                    onInput={linkEvent(this, this.handleUserSettingsBioChange)}
+                    minLength={3}
```

Comment:
Could you add a maxLength here? I'm not sure what a good bio max length should be.... 

If these are twitter style bios, (155 chars) I could see them being useful as a potential user-hover style thing.... But if they're really long "user pages", then I would only want them to be shown on the details page only.

Reply:
Where to put this maxLength constant?

Reply:
If you want you can put it in `utils.ts`, but its fine to just put right there, as this form would get re-used anywhere that check needs to be done anyway.